test/e2e/atlas/serverless_test.go

```diff
@@ -105,12 +105,12 @@ func TestServerless(t *testing.T) {
 		resp, err := cmd.CombinedOutput()
 		req.NoError(err, string(resp))
 
-		var clusters mongodbatlas.ClustersResponse
+		var clusters atlasv2.PaginatedServerlessInstanceDescription
 		err = json.Unmarshal(resp, &clusters)
 		req.NoError(err)
 
 		a := assert.New(t)
-		a.NotEmpty(clusters.Results)
+		a.NotEmpty(clusters)
```

Comment:
leftover from a previous change
```suggestion
		a.NotEmpty(clusters.Results)
```